galaxy/api/internal/views/combined.py

```diff
@@ -0,0 +1,108 @@
+# (c) 2012-2019, Ansible by Red Hat
+#
+# This file is part of Ansible Galaxy
+#
+# Ansible Galaxy is free software: you can redistribute it and/or modify
+# it under the terms of the Apache License as published by
+# the Apache Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# Ansible Galaxy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# Apache License for more details.
+#
+# You should have received a copy of the Apache License
+from rest_framework import views
+from galaxy.main import models
+from rest_framework import response
+from galaxy.api.internal import serializers as internal_serializers
+from galaxy.api import serializers as v1_serializers
+from rest_framework import status
+
+
+class RepoAndCollectionList(views.APIView):
+    def get(self, request):
+        namespace = request.GET.get('namespace', None)
```

Comment:
It's ok to use `request.GET`, but DRF provides a more correctly named alias `request.query_params`. It's preferred over the standard django `request.GET`.
https://www.django-rest-framework.org/api-guide/requests/#query_params

Reply:
Also you may use DRF serializer to parse arguments.